src/components/Checkbox/README.md

```diff
@@ -0,0 +1,64 @@
+#
+
+This demonstrates the use of Checkbox.
+
+```jsx
+const React = require('react');
+const Checkbox = require('./Checkbox').default;
```

Comment:
You can use  es6 import now witht he last version a react-styleguidist  `import React from 'react' ` and  `import Checkbox from 'react-spatial/components/Checkbox' `